.github/issue_template.md

```diff
@@ -0,0 +1,8 @@
+<!---
+Thank you for reporting a problem and suggesting improvements. Please provide the below information to make sure we can effectively deal with the issue reported. For the most precise version information, see the first line of console output or run with --version.
+--->
+
+CBMC version:
+Operating system:
+What behaviour did you expect:
```

Comment:
How did you run CBMC (example code and command line that you ran that would allow us to reproduce the problem)